src/ui/CustomStyleEditor.js

```diff
@@ -1128,15 +1132,15 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
                       <input
                         checked={this.state.styles.hasBullet}
                         className="molsp-chknumbering"
-                        disabled={this.state.styles.styleLevel ? false : true}
+                        disabled={(this.state.styles.styleLevel === 'None' || this.state.styles.styleLevel === undefined) ? true : false}
                         onChange={this.handleBulletPoints.bind(this)}
                         type="checkbox"
                       />
                       Bullet&nbsp;
                       <span>
                         <select
                           className="molsp-fontstyle"
-                          disabled={this.state.styles.hasBullet ? false : true}
+                          disabled={!(this.state.styles.hasBullet) || (this.state.styles.styleLevel === 'None' || this.state.styles.styleLevel === undefined) ? true : false}
```

Comment:
Ambiguity on the '!' intent looks like a possible mistake. The '!' should either be inside the bubble, or the bubble should cover more than 1 expression.